app/package.json

```diff
@@ -41,6 +41,7 @@
     "@coremedia/ckeditor5-dataprocessor-support": "6.1.1-rc.1",
     "@coremedia/ckeditor5-studio-essentials": "6.1.1-rc.1",
     "@coremedia/ckeditor5-symbol-on-paste-mapper": "6.1.1-rc.1",
+    "@coremedia/ckeditor5-richtext-normalizer": "6.1.1-rc.1",
```

Comment:
This package is not about richtext, it is about data-processors in general.

Thus, it perfectly matches `dataprocessor-support` where it was before.

As I understand, that it is important to get rid of irrelevant CKEditor dependencies, we may keep this extra package, but should name it for example:

* `@coremedia/ckeditor5-data-differ`

  This is the main class. It may be misleading, though, as it is not about differencing.

* `@coremedia/ckeditor5-data-normalization`

   I think, this is more to the point. But having this, the central class should not be called `DataDiffer`. See suggestion below.

So, my suggestion would be:

**Package Name:** `@coremedia/ckeditor5-data-normalization`

**Rename:** `DataDiffer` → `DataNormalizer` (and related accordingly, thus, `DataNormalizerMixin`, `isDataNormalizer`, ...).